features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml.j2

```diff
@@ -964,11 +961,20 @@
         </Resource>
 
         <!-- [Organization] Configuration Management API -->
-        <Resource context="(.*)/o/api/server/v1/configs/authenticators(.*)" secured="true" http-method="GET">
```

Comment:
btw, do the same change to resource-access-control-v2.xml as well

Reply:
Done with [bc10eb6](https://github.com/wso2/carbon-identity-framework/pull/5506/commits/bc10eb696770b622413dbc2e9b392e10f432cb76)